modules/mavlink/flight_controller.py

```diff
@@ -351,3 +351,16 @@ def insert_waypoint(
         commands.insert(index, new_waypoint)
 
         return self.upload_commands(commands)
+
+    def send_statustext_msg(
+        self,
+        vehicle: dronekit.Vehicle,
+        message: str,
+        severity: int = mavutil.mavlink.MAV_SEVERITY_INFO,
+    ) -> None:
+        """
+        Sends a STATUSTEXT message to the vehicle.
+        """
+        message_bytes = message.encode("utf-8")
+        msg = vehicle.message_factory.statustext_encode(severity, message_bytes)
```

Comment:
Can you check to make sure there is a target_system and target_component parameter? Please use the ones specified in the example. Also, are you able to test this? Could you write an integration test that just runs this (ie sends a bunch of messages)? You will connect to mission planner just as described in the example, and then see if you can see the messages in mission planner. (Let me know if you need help testing)